pkg/tfgen/parse/extension.go

```diff
@@ -40,12 +44,29 @@ var TFRegistryExtension goldmark.Extender = tfRegistryExtension{}
 type tfRegistryExtension struct{}
 
 func (s tfRegistryExtension) Extend(md goldmark.Markdown) {
+	md.SetRenderer(markdown.NewRenderer())
 	extension.GFM.Extend(md)     // GitHub Flavored Markdown
 	section.Extension.Extend(md) // AST defined sections
 	meta.Extension.Extend(md)    // Support for YAML metadata blocks
 	md.Parser().AddOptions(parser.WithASTTransformers(
 		util.Prioritized(recognizeHeaderAfterHTML{}, 902),
 	))
+	md.Renderer().AddOptions(renderer.WithNodeRenderers(
+		util.Prioritized(tableRenderer{md.Renderer()}, 499),
```

Comment:
Could we add a short version of the commit description as a code comment here? Just like "we do this so the SDKs have docs that make sense"